code/modules/religion/rites.dm

```diff
@@ -366,3 +366,15 @@
 	human2plant.set_species(/datum/species/golem/wood/holy)
 	human2plant.visible_message("<span class='notice'>[human2plant] has been converted by the rite of [name]!</span>")
 	return TRUE
+
+/datum/religion_rites/plantgeist
+	name = "Summon Plantgeist"
+	desc = "Summons forth a holy plantgeist that can heal fellow plant like creatures. Note, it will be dormant till a ghost inhabits it, and it only understands Sylvan."
```

Comment:
```suggestion
	desc = "Summons forth a holy plantgeist that can heal fellow plant-like creatures. Note, it will be dormant till a ghost inhabits it, and it only understands Sylvan."
```